src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
```

Comment:
please write all const with first upperCase letter
`active =` to `Active =`

![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/91946036/9a82357f-a3b9-40b5-937c-a618ff68a1d2)